### ISSUE TITLE
maven build

### ISSUE THREAD

**Issue by joelduerksen** [mSorok/NPlsWeb#9]
is the extra v a typo?

---
compile the project with Maven by running: "**mvnv** clean package" from the project root

**Comment by mSorok** [comment]
No, it depends on the system you use: it will also work with "mvn" command, of course, but it was "mvnv" in my IntelliJ
